core/trino-spi/src/test/java/io/trino/spi/predicate/TestValueSet.java

```diff
@@ -14,18 +14,30 @@
 package io.trino.spi.predicate;
 
 import io.airlift.slice.Slice;
+import io.trino.spi.type.Type;
 import io.trino.spi.type.VarcharType;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.util.List;
+import java.util.Optional;
 
 import static io.airlift.slice.Slices.utf8Slice;
+import static io.trino.spi.predicate.Utils.tryExpandRanges;
```

Comment:
The test should live in the aptly named `TestUtils` class.

(it would belong here if the `tryExpandRanges` method became a `ValueSet` method, which may actually be a good idea.)

Reply:
Moved `tryExpandRanges` to `ValueSet`, thanks!